pandas/tests/series/methods/test_nlargest.py

```diff
@@ -177,7 +177,7 @@ def test_nlargest_nullable(self, any_numeric_ea_dtype):
             arr = np.random.default_rng(2).standard_normal(10)
         arr = arr.astype(dtype.lower(), copy=False)
 
-        ser = Series(arr.copy(), dtype=dtype)
```

Comment:
This could be dangerous

Reply:
Sure sounds good. Added `copy=True` to the constructor